docs/MS Excel/Форматирование ячеек.md

```diff
@@ -0,0 +1,42 @@
+# Форматирование ячеек
+
+##### Форматирование ячеек позволяет изменять внешний вид данных в ячейках, делая таблицы более информативными и профессионально выглядящими. В этой статье мы рассмотрим различные аспекты форматирования ячеек и его возможности.
```

Comment:
[ch-md-error-increment-heading-levels](https://chaberch.github.io/how-to-chaberch/markdown/#ch-md-error-increment-heading-levels)